internal/manager/config.go

```diff
@@ -141,7 +141,7 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 		"Define the rate (in seconds) in which configuration updates will be applied to the Kong Admin API.",
 	)
 	flagSet.Float32Var(&c.ProxyTimeoutSeconds, "proxy-timeout-seconds", dataplane.DefaultTimeoutSeconds,
-		"Define the rate (in seconds) in which the timeout configuration will be applied to the Kong client.",
+		"Define the timeout (in seconds) for all requests to Kong's Admin API",
```

Comment:
```suggestion
		"Sets the timeout (in seconds) for all requests to Kong's Admin API.",
```

Most other flag values use the "indicative mood" rather than "imperative mood" - this comment fixes that.
Same applies to the full stop at the end.